src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -5576,9 +5605,16 @@ switch_status_t config_sofia(sofia_config_t reload, char *profile_name)
 						int v = atoi(val);
 						if (v >= 0) {
 							profile->rtp_timeout_sec = v;
+							sofia_set_media_flag(profile, SCMF_RTP_TIMEOUT_SEC);
 							switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_WARNING,
 											  "rtp-timeout-sec deprecated use media_timeout variable.\n"); 
```

Comment:
@dev-ryanc , please help me identify which variables we should be using instead of `rtp-timeout-sec` as I couldn't spot any `media-timeout` variable being referred here... so we might need to replace the "partner-rtp-timeout-sec" in my changes.